ydb/core/grpc_services/grpc_request_check_actor.h

```diff
@@ -546,22 +546,31 @@ class TGrpcRequestCheckActor
                         << ", database: " << CheckedDatabaseName_
                         << ", user: " << TBase::GetUserSID()
                         << ", from ip: " << GrpcRequestBaseCtx_->GetPeerName());
-            return false;
+            return {false, std::nullopt};
         }
 
         const ui32 access = NACLib::ConnectDatabase;
         const auto& parsedToken = TBase::GetParsedToken();
         if (parsedToken && SecurityObject_->CheckAccess(access, *parsedToken)) {
-            return false;
+            return {false, std::nullopt};
         }
 
         Counters_->IncDatabaseAccessDenyCounter();
 
         if (!AppData()->FeatureFlags.GetCheckDatabaseAccessPermission()) {
-            return false;
+            return {false, std::nullopt};
         }
 
-        return true;
+        const TString error = "No permission to connect to the database";
+        LOG_INFO_S(TlsActivationContext->AsActorContext(), NKikimrServices::GRPC_SERVER, 
+            "AUDIT: "
```

Comment:
This is not an audit message, so `AUDIT:` mark should not be used.

`AUDIT:` mark is only present in common log messages that duplicate (some) audit log messages and that are left in the code for temporary backward compatibility with the audit log consumers that lagging behind in adopting separate audit log stream.
As a rule, `AUDIT:` should not be added to the newly introduced common log messages.

Reply:
a1852a1af71dfd3b7b036f990283450aef94db6a